spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3926,6 +3926,7 @@ def insert_text(event):
         elif key == Qt.Key_Insert and not shift and not ctrl:
             self.setOverwriteMode(not self.overwriteMode())
         elif key == Qt.Key_Backspace and not shift and not ctrl:
+            self.document_did_change()

```

Comment:
I think this should be called at the end of the condition block, or else it does nothing.

Reply:
You're totally right. @andfoy, please do that change.